frontend/app/routes/$lang+/_public+/apply+/$id+/adult-child/review-adult-information.tsx

```diff
@@ -343,6 +357,7 @@ export default function ReviewInformation() {
       try {
         const response = captchaRef.current.getResponse();
         formData.set('h-captcha-response', response);
+        formData.set('_action', FormAction.Submit);
```

Comment:
Is it needed? since the buttons have the name `_action` with their value. It should be pushed by default.

Reply:
It's because of the `handleSubmit`. We're preventing the default behaviour for hCaptcha.  I found that I have to manually set it here to pass it along to the action, otherwise the action will error when zod tries to parse value

Reply:
To make it works, the submit event's submitter must be pass to `new FormData`. `FormEvent` must be changed to `SyntheticEvent<HTMLFormElement, SubmitEvent>`.

```
function handleSubmit(event: SyntheticEvent<HTMLFormElement, SubmitEvent>) {
    event.preventDefault();
    const formData = new FormData(event.currentTarget, event.nativeEvent.submitter);
```